config/workers.coffee

```diff
@@ -8,6 +8,11 @@ module.exports = (KONFIG, options, credentials) ->
   nodeProgram = if options.watchNode then './watch-node' else 'node'
 
   workers =
+    nginx:
+      group: 'webserver'
+      supervisord:
+        command: 'nginx -c %(ENV_KONFIG_PROJECTROOT)s/nginx.conf'
```

Comment:
s/nginx.conf/deployment\/generated_files/nginx.conf

also could you remove the `fs.writeFileSync "./nginx.conf", KONFIG.nginxConf` line from configure? any idea why we are duplicating it?